frappe/public/js/frappe/views/reports/report_view.js

```diff
@@ -799,7 +802,10 @@ frappe.views.ReportView = class ReportView extends frappe.views.ListView {
 			const field = [col, doctype];
 			this.fields.push(field);
 			this.refresh();
-			frappe.show_alert(__('Also adding the status dependency field {0}', [field[0].bold()]));
+			const field_label = frappe.meta.get_label(doctype, field[0]);
```

Comment:
Labels are not necessarily unique
Either we should use only fieldnames or fieldnames + label
`Label (fieldname)`